Submitty/Submitty
# [Refactor:InstructorUI] Change Course Materials Select Widget

### What is the current behavior?
When I created the folder recommendations list for the course materials upload and edit button, I couldn't figure out how to replicate the select2 widget we use on the Admin Gradeable Config page. Since then @taorif25 replicated this widget on the exceptions page. To match the styling of those two pages, this PR updates the widget to match the other pages.

### What is the new behavior?
The widget behaves the same, but looks like the other pages.

### How to test?
Try uploading files and moving files in Course Materials and see if anything breaks.

### Images
Before:
![image](https://github.com/Submitty/Submitty/assets/51209504/17cba6a8-8743-4b8d-8fd7-b97477c87568)

![image](https://github.com/Submitty/Submitty/assets/51209504/fb85bddb-fcde-45db-bd5f-4c1d700f082a)

After:
![image](https://github.com/Submitty/Submitty/assets/51209504/ffab555b-7089-41c7-af65-adc8b84405d5)

![image](https://github.com/Submitty/Submitty/assets/51209504/df3e9e2a-72b8-4a26-813a-e4249e200757)
